tests/cephfs/cephfs_bugs/test_mds_laggy_thrown_out.py

```diff
@@ -0,0 +1,147 @@
+        )
+
+        fuse_mount_dir = "/mnt/fuse_" + "".join(
+            secrets.choice(string.ascii_uppercase + string.digits) for i in range(5)
+        )
+        fs_util.fuse_mount(
+            [client1],
+            fuse_mount_dir,
+            new_client_hostname="admin",
+            extra_params=f" --client_fs {fs_name}",
+        )
+        client1.exec_command(
+            sudo=True,
+            cmd=f"cd {fuse_mount_dir};"
+            f"mkdir subdir;"
+            f"dd if=/dev/urandom of=subdir/sixmegs bs=1M conv=fdatasync count=6 seek=0;",
+        )
+
+        standby_reply_mdss = fs_util.get_standby_replay_mdss(client1, fs_name=fs_name)
+        standby_reply_mdss_objs = [
+            ceph_cluster.get_node_by_hostname(i.split(".")[1])
+            for i in standby_reply_mdss
+        ]
+        standby_reply_mdss_objs[0].upload_file(
+            sudo=True,
+            src="tests/cephfs/cephfs_bugs/network_disconnect.sh",
+            dst=f"/root/{file}",
+        )
+        out, rc = client1.exec_command(
+            sudo=True,
+            cmd=f"ceph fs status {fs_name}",
+        )
+        log.info(out)
+        standby_reply_mdss_objs[0].exec_command(
+            sudo=True,
+            cmd=f"bash /root/{file} {standby_reply_mdss_objs[0].ip_address} 10",
+        )
+        out, rc = client1.exec_command(
+            sudo=True,
+            cmd=f"ceph fs status {fs_name}",
+        )
+        log.info(out)
+        standby_reply_mdss_10 = fs_util.get_standby_replay_mdss(
+            client1, fs_name=fs_name
+        )
+        standby_reply_mdss_objs_10 = [
+            ceph_cluster.get_node_by_hostname(i.split(".")[1])
+            for i in standby_reply_mdss_10
+        ]
+        if standby_reply_mdss != standby_reply_mdss_10:
+            raise CommandFailed(
+                f"Standby nodes has changed\n Before : {standby_reply_mdss} \n After : {standby_reply_mdss_objs_10}"
+            )
+        standby_reply_mdss_objs[0].exec_command(
+            sudo=True,
+            cmd=f"bash /root/{file} {standby_reply_mdss_objs[0].ip_address} 60",
+        )
+        out, rc = client1.exec_command(
+            sudo=True,
+            cmd=f"ceph fs status {fs_name}",
+        )
+        log.info(out)
+        standby_reply_mdss_60 = fs_util.get_standby_replay_mdss(
+            client1, fs_name=fs_name
+        )
+
+        if standby_reply_mdss == standby_reply_mdss_60:
+            raise CommandFailed(
+                f"Standby nodes did not change\n Before : {standby_reply_mdss} \n After : {standby_reply_mdss_60}"
+            )
+
+        return 0
+
+    except CommandFailed as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
+    except Exception as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
+    finally:
+        client1.exec_command(sudo=True, cmd=f"umount {fuse_mount_dir}", check_ec=False)
```

Comment:
As practice, Can we reuse method "client_clean_up" for all tests to cleanup mounts and directories instead of all tests having to cleanup explicitly ?